### PR TITLE
Use ansible galaxy

### LINKED ISSUE #8: We need change to Ansible galaxy
To project improvement will need use ansible galaxy to declare our ansible roles.

### PR DESCRIPTION
**Pull request by lborguetti**
This PR closes #8 and add packer-fmt utils.

### CODE REVIEW

**Review by evaldofelipe — APPROVED** [review]
Awesome code! Nothing to report! Like a glove! 👌

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ansible-azure.cache
 *.retry
 vault_password_file
+ansible/roles/
```
Comment (OctavioBR) [inline .gitignore]: I don't think this line is necessary, by default (without `-p` or different `roles_path` config) `ansible-galaxy install` saves required roles into .ansible home folder, outside this repo. I even prefer letting this path tracked when we may start another role from scratch, without publishing it into other git repo or galaxy... What you think? 😃

**Review by OctavioBR — APPROVED** [review]
Great job 👍

File: .gitignore (hunk 1 above)
Comment (lborguetti) [reply .gitignore]: Nice tips @OctavioBR :clap: 

I'll change and test this approach.

Comment (lborguetti) [reply .gitignore]: @OctavioBR  Changed. 

Thanks dude.

**Pull request merged**